### ISSUE TITLE
Change/move wording about Tax deductible purchases.

### ISSUE THREAD

**Issue by paulstroot** [ocupop/comission-shopify-hydrogen#194]
Bills request to move the museum-generated text about a purchase being tax deductible from lower down on the page so that it is directly below the blue "Buy Now" Button.

Some questions to work out:

1. The text lower down on the page is currently made up of two parts, the first part is programitally created by inserting the museum name into `"Your purchase is a charitable donation to {Museum Name}, and is tax-deductible within the guidelines of U.S. law."`
2. The second part is written by the museum
![image](https://user-images.githubusercontent.com/52838369/236868894-afced271-5c3f-4385-a8cc-509d5f99b203.png)

Below is a screenshot of where the text will be moved to:
![image](https://user-images.githubusercontent.com/52838369/236869173-51c3b411-db88-4662-964f-8b6d4524c0b1.png)

**2 Questions:**

1. There is no guarantee that the museum will write text that includes verbiage related to the purchase being tax-deductible. Would we like to programaticallly insert the part about the purchase being tax deductible like we do below? Or count on the museum including that information in their copy?
2. How will we change the section lower down on the page "Collect your DAB and support the arts"?

**Comment by ZiggyAtl** [comment]
Can we just remove the second piece further down the page please. Then under the Buy Now button (thanks for doing that btw) have;

Collect your DAB and support its cause [in same type height as the Artist name above]
[insert section written by the museum abut where the money is being spent]

I know the above is less automated and will require me to make sure the museum include the piece about being tax deductible. But there may be times when the actual recipient of the benefit might not be the museum (as may be with this first artwork). So i'll just make a note to check that is included before I approve the piece for CoMissioning. That's also why I changed the title to "...support its cause" as it may not be the arts. In the case of this first one it may be going to support victims of human trafficking.

**Comment by paulstroot** [comment]
All Set @ZiggyAtl . Here is an example artwork page for you to review:
https://store.comission.art/products/heels-out

**Comment by paulstroot** [comment]
Yes. I reduced the amount if space in half. There should now be equal space between all of the sections.

![image](https://github.com/ocupop/comission-shopify-hydrogen/assets/52838369/fdd1f575-1412-4e9e-bc37-3de0002fc942)

**Comment by ZiggyAtl** [comment]
Thank you
